src/hotspot/share/opto/predicates.cpp

```diff
@@ -323,7 +323,7 @@ Opaque4Node* TemplateAssertionPredicateExpression::clone(const TransformStrategy
   auto is_opaque_loop_node = [](const Node* node) {
     return node->is_Opaque1();
   };
-  DataNodesOnPathsToTargets data_nodes_on_path_to_targets(TemplateAssertionPredicateExpression::maybe_contains,
+  DataNodesOnPathsToTargets data_nodes_on_path_to_targets(TemplateAssertionPredicateExpressionNode::valid_opcode,
```

Comment:
Moved `TemplateAssertionPredicateExpression::maybe_contains()` to `TemplateAssertionPredicateExpressionNode::valid_opcode()` which suited better.